components/containers/Admin/Organizations/CreateOrganization.tsx

```diff
@@ -43,11 +43,33 @@ export const CreateOrganization = (): React.ReactElement => {
           <Label htmlFor="orgNameEn" required={true}>
             Organization Name (English)
           </Label>
-          <TextInput id="orgNameEn" name="orgNameEn" type="text" required={true}></TextInput>
+          <TextInput
+            id="orgNameEn"
```

Comment:
IMO: It may make more sens to stick to one style  for consistency accros the code base. 

`id="orgNameEn"'     name="orgNameEn`"  this is more consistent with javascript code.

Further in the code you will encounter a different syntax like so.
These below look more like html4/5-like attributes or in html.

```
id="input-type-text"
name="input-type-text"
```

Reply:
Fair enough. This isn't something that I introduced, but PRs will often reveal details that may have slipped though before.